sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/QuerySamples.cs

```diff
@@ -74,15 +76,99 @@ await foreach (TimeSeriesPoint point in humidityEventsQueryAnalyzer.GetResultsAs
+            };
+
+            timeSeriesTypes.Add(new TimeSeriesType("TemperatureSensor", variables) { Id = "TemperatureSensorTypeId" });
+
+            Response<TimeSeriesTypeOperationResult[]> createTypesResult = await client
+                .Types
+                .CreateOrReplaceAsync(timeSeriesTypes)
+                .ConfigureAwait(false);
+
+            if (createTypesResult.Value.First().Error != null)
+            {
+                Console.WriteLine($"\n\nFailed to create a Time Series Insights type. " +
+                    $"Error Message: '{createTypesResult.Value.First().Error.Message}.' " +
+                    $"Code: '{createTypesResult.Value.First().Error.Code}'.");
+            }
+            else
+            {
+                // Get the Time Series instance and replace its type with the one we just created
+                Response<InstancesOperationResult[]> getInstanceResult = await client.Instances.GetAsync(new List<TimeSeriesId> { tsId });
+                if (getInstanceResult.Value.First().Error != null)
+                {
+                    Console.WriteLine($"\n\nFailed to retrieve Time Series instance with Id '{tsId}'. " +
+                        $"Error Message: '{getInstanceResult.Value.First().Error.Message}.' " +
+                        $"Code: '{getInstanceResult.Value.First().Error.Code}'.");
+                }
+                else
+                {
+                    TimeSeriesInstance instanceToReplace = getInstanceResult.Value.First().Instance;
+                    instanceToReplace.TypeId = createTypesResult.Value.First().TimeSeriesType.Id;
+                    Response<InstancesOperationResult[]> replaceInstanceResult = await client.Instances.ReplaceAsync(new List<TimeSeriesInstance> { instanceToReplace });
+                    if (replaceInstanceResult.Value.First().Error != null)
+                    {
+                        Console.WriteLine($"\n\nFailed to retrieve Time Series instance with Id '{tsId}'. " +
+                            $"Error Message: '{replaceInstanceResult.Value.First().Error.Message}.' " +
+                            $"Code: '{replaceInstanceResult.Value.First().Error.Code}'.");
+                    }
+                    else
+                    {
+                        // Now that we set up the instance with the property type, query for the data
+                        #region Snippet:TimeSeriesInsightsSampleQuerySeries
+                        Console.WriteLine($"\n\nQuery for temperature series in Celsius and Fahrenheit over the past 10 minutes. " +
+                            $"The Time Series instance has a type that has predefined numeric variable that represents the temperature " +
+                            $"in Celsuis, and a predefined numeric variable that represents the temperature in Fahrenheit.\n");
+
+                        DateTimeOffset endTime = DateTime.UtcNow;
+                        DateTimeOffset startTime = endTime.AddMinutes(-10);
```

Comment:
can use a comment here for 10 minutes.

Reply:
Actually there is a comment on top that says past 10 mins. Do you think it's still necessary to have a separate comment here?